Mobile/lib/Components/Navigation.dart

```diff
@@ -7,6 +7,7 @@ import 'package:mobile/NavigationComponents/DrawerItem.dart';
 import 'package:mobile/NavigationComponents/NavigationAppBar.dart';
 import 'package:mobile/NavigationComponents/NavigationDrawer.dart';
 import 'package:mobile/config/app_pages.dart';
+import 'package:mobile/config/task_list_types.dart';
```

Comment:
???

Reply:
Jo den bliver brugt, det er for at importere TasklistType, jeg fjernede den fra Tasklist component, da jeg ikke synes det giver mening at importere et helt component bare for at få en enum / type.